### PR TITLE
[ADD] sale_force_whole_invoiceability: New module

### PR DESCRIPTION
**Pull request by chienandalu**
cc @Tecnativa TT27917

### CODE REVIEW

File: sale_force_whole_invoiceability/README.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    :target: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_force_whole_invoiceability
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/167/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows sale managers to force all the lines to invoice independently
+of the products invoive policy and the delivered quantities.
```
Comment (pedrobaeza) [inline sale_force_whole_invoiceability/README.rst]:
```suggestion
This module allows to the sale managers to force all the lines to invoice independently
of the invoice policy and the delivered quantities of the products.
```

**Review by pedrobaeza — APPROVED** [review]
Great reuse of the core!

File: sale_force_whole_invoiceability/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Sale Force Whole Invoiceability",
+    "version": "12.0.1.0.0",
+    "category": "Sales",
+    "author": "Tecnativa,"
+              "Odoo Community Association (OCA)",
+    "website": "https://www.github.com/OCA/partner-contact",
```
Comment (rousseldenis) [inline sale_force_whole_invoiceability/__manifest__.py]:
```suggestion
    "website": "https://www.github.com/OCA/sale-workflow",
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: sale_force_whole_invoiceability/README.rst (hunk 1 above)
Comment (chienandalu) [reply sale_force_whole_invoiceability/README.rst]: done :+1:

File: sale_force_whole_invoiceability/__manifest__.py (hunk 2 above)
Comment (chienandalu) [reply sale_force_whole_invoiceability/__manifest__.py]: ups, thanks!

Comment (chienandalu) [reply sale_force_whole_invoiceability/__manifest__.py]: fixed

**Review by rousseldenis — APPROVED** [review]

File: sale_force_whole_invoiceability/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+To use this module, you need to:
+
+#. Make a sale order with products with a `delivered quantities` invoice policy.
+#. Confirm the order.
+#. If you try to invoice the order, you can't as the quantities aren't delivered.
+#. But if you are a `Sales Manger`, you have a new button *Force Invoice Policy*
```
Comment (marcelsavegnago) [inline sale_force_whole_invoiceability/readme/USAGE.rst]:
```suggestion
#. But if you are a `Sales Manager`, you have a new button *Force Invoice Policy*
```

Comment (chienandalu) [reply sale_force_whole_invoiceability/readme/USAGE.rst]: thanks @marcelsavegnago ! fixed

**Review by marcelsavegnago — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu It seems Travis has gone to red in the last push.

**Comment by chienandalu** [comment]
Looks like a test from `sale_order_lot_generator` is failing on a missing attribute. The weird thing is that it didn't fail until now: https://travis-ci.com/github/OCA/sale-workflow/jobs/479997013#L2637

**Comment by pedrobaeza** [comment]
#1446

**Comment by chienandalu** [comment]
Rebased and fixed

**Comment by chienandalu** [comment]
Forward ported https://github.com/OCA/sale-workflow/pull/1470

**Pull request merged**